Oleanparser/Object.lean

```diff
@@ -75,4 +75,4 @@ unsafe def reprCore : Obj → ReprM Format
     return newDeclId
 
 unsafe instance : Repr Obj where
-  reprPrec o := Repr.addAppParen o.reprCore.run
+  reprPrec o := Repr.addAppParen "Obj _"
```

Comment:
Please revert this accidental change here.  The `Demo.lean` file contains an end-to-end demo, the last `#eval` there should print the contents of `testData`.

Reply:
I had to fix OfNat Any, but now it works. Hope to be of assistance in this project.

Reply:
Thanks!

Reply:
Oops, apparently I was a bit too quick with merging.  The OfNat instance you added wasn't quite what I expected (it converted every natural number to `()`).  I've changed it `⟨_, n⟩` which was the previous behavior.